code/game/gamemodes/cult/cultify/obj.dm

```diff
@@ -128,12 +130,17 @@
 	..()
 
 /obj/structure/table/cultify()
-	// Make it a wood-reinforced wooden table.
-	// There are cult materials available, but it'd make the table non-deconstructable with how holotables work.
-	// Could possibly use a new material var for holographic-ness?
-	material = get_material_by_name("wood")
-	reinforced = get_material_by_name("wood")
+	if(material == "cult" || reinforced == "cult")
```

Comment:
We should really have material defines. Not something you need to do for this PR though.